rebasehelper/tests/test_utils.py

```diff
@@ -527,7 +527,7 @@ def test_all_packages_installed_one_non_existing(self):
 class TestMacroHelper(BaseTest):
 
     def test_get_macros(self):
-        rpm.addMacro('test_macro', 'test_macro value')
+        rpm.addMacro('test_macro', 'test_macro value')  # pylint: disable=no-member
```

Comment:
Dont guess and check older travis builds where it was not working even the module was ignored.